pkg/operator/ceph/cluster/mgr/spec.go

```diff
@@ -178,13 +179,11 @@ func (c *Cluster) makeSetServerAddrInitContainer(mgrConfig *mgrConfig, mgrModule
 
 	container := v1.Container{
 		Name: "init-set-" + strings.ToLower(mgrModule) + "-server-addr",
-		Command: []string{
-			"ceph",
-		},
-		Args: append(
-			controller.AdminFlags(c.clusterInfo),
-			cfgSetArgs...,
-		),
+		Command: append([]string{
+			"/bin/bash",
```

Comment:
Why not just call `timeout` directly? Then you can use arguments without joining.

Reply:
Yes that worked better, thanks